xml/operations-opsconsole-en-alarm_definitions.xml

```diff
@@ -158,34 +107,30 @@
      </listitem>
      <listitem>
       <para>
-       <guimenu>Relational Operator</guimenu>: specifies the operator for how
+       <guimenu>Comparator</guimenu>: specifies the operator for how
        you want the alarm to trigger.
       </para>
      </listitem>
      <listitem>
       <para>
-       <guimenu>Value</guimenu>: determines the numeric threshold associated
+       <guimenu>Threshold</guimenu>: determines the numeric threshold associated
        with the operator you specified.
       </para>
      </listitem>
     </itemizedlist>
    </listitem>
    <listitem>
     <para>
-     Select whether you want to apply the expressions
-     <guimenu>Individually</guimenu> or <guimenu>As a group</guimenu> to the
-     previously selected dimension(s).
+     <guimenu>Match By</guimenu> (optional): group results by a specific
+     dimension that is not part of the Dimension(s) solution.
     </para>
    </listitem>
    <listitem>
     <para>
-     When done creating the alarm, click <guimenu>Create Alarm</guimenu>.
+     Press <guimenu>Create Alarm Definition</guimenu>, to save the changes and
```

Comment:
Same as above press->click.
You also reversed the sentence order, so it is worse now: It should be "goal, action" and not "action, goal". (This ensures people know what they're doing even if they are trying to read only parts of sentences.)